package.json

```diff
@@ -20,7 +20,8 @@
     "lint": "eslint src",
     "check": "flow check",
     "build": "gulp dist",
-    "start": "node ./scripts/start.js",
+    "start": "gulp dist && node ./scripts/start.js",
```

Comment:
rather than redefine `start` and create a `start-norebuild` option, leave it alone and create a `start-rebuild` option. That will maintain `start`s functionality and offer the new functionality without breaking existing applications.